scripts/hallucinate_fragalysis.py

```diff
@@ -0,0 +1,589 @@
+"""
+Build library of ligands from a dataset of holo crystal structures docked to a
+different dataset of apo structures.
+"""
+import argparse
+from glob import glob
+import itertools as it
+import multiprocessing as mp
+from openeye import oechem, oedocking
+import os
+import pandas
+import pickle as pkl
+import re
+import sys
+
+sys.path.append(f"{os.path.dirname(os.path.abspath(__file__))}/../")
```

Comment:
We shouldn't need to do this, what are you trying to accomplish with this?

Reply:
added issue here: https://github.com/choderalab/covid-moonshot-ml/issues/48